### PR TITLE
Add secondary peerjs server as fallback and more STUN providers

### PR DESCRIPTION
**Pull request by yemel**
* Add more STUN servers
* Use a list of peerJS server on config
* Fallback strategy on provider
* Add peerJS server index on joining URL
* Improve error message con stream creation

### CODE REVIEW

File: app/client/controller.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ angular.module('streamium.client.controller', ['ngRoute'])
 
 .config(['$routeProvider',
   function($routeProvider) {
-    $routeProvider.when('/join/:streamId', {
+    $routeProvider.when('/join/:serverIndex/:streamId', {
```
Comment (maraoz) [inline app/client/controller.js]: hmmm this is backwards incompatible. is it possible not to include this in the URL? Maybe try all peerjs servers in roundrobin?

File: app/components/peer.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    ret.port = config.peerServers[index].port;
+    ret.index = index;
+    return ret;
+  }
+
+  return {
+    primary: getPeerJSConfig(config, 0),
+    secondary: getPeerJSConfig(config, 1),
+    get: function(index) {
+      console.log('INDEX> ', index);
```
Comment (maraoz) [inline app/components/peer.js]: console.log

**Comment by yemel** [comment]
Updated

**Comment by maraoz** [comment]
great work!

**Pull request merged**